collect_app/src/main/java/org/odk/collect/android/configure/qr/AppConfigurationGenerator.kt

```diff
@@ -25,6 +27,20 @@ class AppConfigurationGenerator(
         }.toString()
     }
 
+    fun getAppConfigurationAsJsonWithGoogleDriveDetails(googleAccount: String?): String {
+        val generalSettings = JSONObject().apply {
+            put(GeneralKeys.KEY_PROTOCOL, Collect.getInstance().getString(R.string.protocol_google_sheets))
```

Comment:
I don't like using resources here and `Collect.getInstance()`. I also ran into a similar problems when I had to use a value from `untranslated.xml`. Wouldn't it be better to move those untranslated strings to a java/kotlin class so that we don't have to read them from resources?

Reply:
Feels a little out of scope but it's bothered me before so I introduced the values in `GeneralKeys`. This follows the pattern other values have. It does mean the corresponding array (arrays.xml) has a value that's separately defined but I think that's ok.

Would be good to carefully review 4722ece to make sure I didn't use the wrong value anywhere.